### ISSUE TITLE
Federated sources support inc googlesheets

### ISSUE THREAD

**Issue by MarkEdmondson1234** [MarkEdmondson1234/bigQueryR#8]
https://cloud.google.com/blog/big-data/2016/05/bigquery-integrates-with-google-drive

**Comment by Phippsy** [comment]
Hi @MarkEdmondson1234 - just checking, I'm guessing federated source access isn't possible at the moment? 
I tried enabling all the scopes for drive, cloud platform & bigquery then listing them in `googleAuthR.scopes.selected`, per [this article](https://stackoverflow.com/questions/40731823/encountered-an-error-while-globbing-file-pattern-error-when-using-bigquery-api) but can't get it working - I get an error:

```
Request Status Code: 403
Error : JSON fetch error: Access Denied: BigQuery BigQuery: Permission denied while globbing file pattern.
```

Not a big deal, just curious. Thanks as always!

**Comment by MarkEdmondson1234** [comment]
No, not yet.  It would require modifying [this function](https://github.com/cloudyr/bigQueryR/blob/master/R/tables.R#L165) to accept  an `externalDataConfiguration` object as [specced here](https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#resource).
